### DISCUSSION TITLE
Welcome to dominate Discussions!

### DISCUSSION THREAD

**Discussion by Knio** [Knio/dominate#138] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mrodent** [comment]
Hi, hope this is the right place to ask this.

This module is great, very useful and a way to avoid horrible spaghetti code to build HTML.

I was just wondering, is there a way to "render in two"?  

By this I mean, say you build a SPAN tag, and then you actually only want the HTML text of the opening tag ('`<SPAN attr1="..." attr2=""...>`'), or of the closing tag (always '`</SPAN>`'), is there a way to do that? It may seem trivial, but now I've discovered dominate I just hate having to involve strings with unsightly pointy brackets in my code (and even add attributes in a "stringy" way).
